cpu/stm32/periph/flashpage.c

```diff
@@ -185,17 +219,35 @@ void flashpage_write(void *target_addr, const void *data, size_t len)
     stmclk_enable_hsi();
 #endif
 
+#ifdef FLASH_ACR_DCEN
+    if (FLASH->ACR & FLASH_ACR_DCEN) {
+        FLASH->ACR &= ~FLASH_ACR_DCEN;
+    }
+#endif
+#ifdef FLASH_ACR_ICEN
+    if (FLASH->ACR & FLASH_ACR_ICEN) {
+        FLASH->ACR &= ~FLASH_ACR_ICEN;
+    }
+#endif
```

Comment:
What do those blocks do?

Reply:
Data and instruction caches, clarified with a comment